specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/applicationGateway.json

```diff
@@ -2219,6 +2715,21 @@
           "$ref": "#/definitions/ApplicationGatewayAutoscaleConfiguration",
           "description": "Autoscale Configuration."
         },
+        "privateLinkResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGatewayPrivateLinkResource"
+          },
+          "description": "Private Link resources configured on application gateway."
+        },
```

Comment:
the private link resources typically aren't exposed on the resource, just the connections.  Why differ from the normal implementation here?

Reply:
@pilor , after discussion meeting that we had, I will change few stuff .
The plan decided is as follows
1. Remove the PrivateLinkResource from Application gateway. No set will be supported for any service for Private link resources like other services and neither it will show up on Service object model

2. Configuration that is needed for Application Gateway and other services will be under a differect name "privateLinkConfiguration"

3. Get .../applicationgateways/privatelinkresources will just be a API call whose reply will be in line with other services.

I will target the changes now in may release and hence closing the PR for now and will rase a new one 

CC : @MyronFanQiu 

Will send email separately just to get signoffs